parsons/databases/table.py

```diff
@@ -164,3 +165,70 @@ def truncate(self):
 
         self.db.query(f"TRUNCATE TABLE {self.table}")
         logger.info(f"{self.table} truncated.")
+
+    def dedup_table(
+        self,
+        order_by_column_name=None,
+        order_by_direction=None,
+        cascade=False,
+        columns_to_ignore=None,
+    ):
+        """
+        Description:
+            This function re-creates a deduped version of a table by grabbing
+            all columns and inserting those into a partition statement for
+            row_number().
+        Args:
```

Comment:
All of these args should be documented as Optional. (That's something we haven't always done a great job of following, especially in the older code.)

Parsons doesn't follow the Google styleguide exactly, but I like to generally follow these: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html